src/compiler/types.ts

```diff
@@ -4167,6 +4169,7 @@ export interface FlowReduceLabel extends FlowNodeBase {
     antecedent: FlowNode;
 }
 
+/** @internal */
 export type FlowType = Type | IncompleteType;

```

Comment:
Probably I should have noticed this in #58036, but we already make public all of the other types.

Reply:
This is used by nobody: https://github.com/search?q=%2F%5Cbts%5C.FlowType%2F+%28language%3ATypeScript+OR+language%3AJavaScript%29&type=code